lib/tpm/key_attestation.rb

```diff
@@ -86,7 +86,13 @@ def trustworthy?
     def trust_store
       @trust_store ||=
         OpenSSL::X509::Store.new.tap do |trust_store|
-          root_certificates.each { |root_certificate| trust_store.add_cert(root_certificate) }
+          root_certificates.each do |root_certificate|
+            begin
+              trust_store.add_cert(root_certificate)
+            rescue OpenSSL::X509::StoreError => e
+              raise unless e.message == "cert already in hash table"
```

Comment:
I was thinking, just to be more robust, of doing:

```suggestion
              raise unless e.message =~ /cert already in hash table/i
```